src/settings/_typography-responsive.scss

```diff
@@ -26,11 +26,11 @@ $govuk-typography-scale: (
   80: (
     null: (
       font-size: 53px,
-      line-height: 55px
+      line-height: 1.0377358491 // 55px
```

Comment:
I do wonder if these would have been clearer if written as:

```scss
line-height: 55px / 53px;
```

Thoughts?

Reply:
Alternatively, we could have left these as pixel values in the font-map and calculated them in the typography-responsive mixin.

Reply:
I tried various ways of doing this calculation but kept running into issues...Should probably refactor so it's more "automatic" / less prone to typos. If we haven't released can we carry on?

Reply:
If we don't expect this to change often, might be worth doing less here?

Reply:
Yep, happy with this. I've been playing locally with this too – looks like our font-size and line-height are ending up as strings somehow, which means using them in calculations doesn't work. It'd be great to be able to define everything in pixels and everything 'just work', but happy with this for now 👍

Reply:
I kind of got something interesting when spitting it out as a string if we change the output to `font: 16 / 20` and not define line height independently... but this didn't play very nicely with the other PR that needed two font-size declarations.